astroquery/sdss/core.py

```diff
@@ -1072,5 +1075,55 @@ def _get_crossid_url(self, data_release):
         self._last_url = url
         return url
 
+    def view_in_sdss_navigate(self, sc):
+        """
+        To view "Navigate" given the SkyCoord sc of the point of interest.
+
+        Parameters
+        ----------
+        sc: SkyCoord of a point
+
+        Returns
+        -------
+        Response: Open a browser tab in the SDSS "Navigate" interface
+
+        """
+        if isinstance(sc, SkyCoord):
+            if sc.shape == ():
```

Comment:
Is this the best way to test for a scalar coordinate?  It might be, but it looks a little awkward as the intent isn't clear from the test here.  Maybe `sc.ndim == 0`?

Reply:
Yup, changed to `sc.dnim == 0`.